crates/biome_aria/src/roles.rs

```diff
@@ -1296,6 +1310,51 @@ impl<'a> AriaRoles {
 
         role_candidate.concepts_by_role()
     }
+
+    pub fn is_not_static_element(
+        &self,
+        element_name: &str,
+        attributes: &FxHashMap<String, Vec<String>>,
+    ) -> bool {
+        if match element_name {
+            // embedded content
+            // ref: https://html.spec.whatwg.org/multipage/semantics.html#embedded-content
+            "canvas" | "embed" | "iframe" | "video" | "audio" => true,
+            // metadata content
+            // ref: https://html.spec.whatwg.org/multipage/semantics.html#document-metadata
+            "meta" | "link" | "base" | "title" | "basefont" | "head" => false,
+            // scripting content
+            // ref: https://html.spec.whatwg.org/multipage/semantics.html#scripting-content
+            "script" | "noscript" | "template" | "style" => false,
+            // No corresponding role
+            "dl" | "label" | "legend" | "ruby" | "pre" | "figcaption" | "br" => true,
+            _ => false,
+        } {
+            return true;
+        }
+
+        // if the element has a interactive role, it is considered interactive.
+        let role_name = if let Some(role) = attributes.get("role") {
+            if let Some(r) = role.first() {
+                self.get_role(r)
+            } else {
+                None
+            }
+        } else {
+            self.get_implicit_role(element_name, attributes)
+        };
+
+        if let Some(role) = role_name {
+            match role.type_name() {
+                "biome_aria::roles::PresentationRole" | "biome_aria::roles::GenericRole" => {
+                    return false
+                }
+                _ => return true,
+            }
+        }
+
+        false
```

Comment:
This can be more simple (if this works)
```suggestion
    let role_name = attributes
        .get("role")
        .and_then(|role| role.first())
        .map(|r| self.get_role(r))
        .or_else(|| self.get_implicit_role(element_name, attributes));
    
    match role_name.map(|role| role.type_name()) {
        Some("biome_aria::roles::PresentationRole" | "biome_aria::roles::GenericRole") => false,
        Some(_) => true,
        None => false,
    }
```

Reply:
@unvalley 
thanks,
a part of your suggestion code do not work, so i fixed!

[7063f9b](https://github.com/biomejs/biome/pull/2981/commits/7063f9b2e9476b154e478c65d9233b014aea145d)